client/app/widgets/Notifications/index.jsx

```diff
@@ -11,124 +11,128 @@ export type Props = {
 };
 
 export type State = {
-  notifications: string,
-  alreadyMounted: boolean,
-  open: boolean,
+  notifications?: string,
+  alreadyMounted?: boolean,
+  open?: boolean,
   modalKey?: string,
   signedInKey?: number,
 };
 
-export class Notifications extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = { notifications: '', alreadyMounted: false, open: false };
-  }
+export const Notifications = ({
```

Comment:
So I think you're getting those errors you identified because of how you exported things.

You'll want to do something like [this](https://github.com/ifmeorg/ifme/blob/master/client/app/components/Modal/index.jsx#L174).

Reply:
Ok I am changing this